it/README.md

```diff
@@ -0,0 +1,81 @@
+# Licenze Creative Commons Licenses per progetti GitHub
+
+[![Main](https://img.shields.io/badge/main%20language-EN-blue)](/../../)
+
+In questo repo troverai dei modi facili di applicare le licenze Creative Commons ai repository attraverso il linguaggio Markdown.
+
+> **ATTENZIONE:**
+> Tu non dovrai usare le licenze Creative Commons per i programmi.
+> Usa le licenze Free Software per questi tipi di repository, come ad esempio GPL, BSD, ecc.
+> Le licenze Creative Commons sono pensate solo per le opere intellettuali.
+
+Il modo più semplice per trovare la giusta licenza CC per il tuo progetto è il
+[sito Creative Commons](https://creativecommons.org/choose/?lang=it).
+
+Qui vi presento due tipi di licenze CC:
+
+* [Creative Commons Attribution 4.0 International](#cc-attribution-40-international)
```

Comment:
I think the names of the licenses should be also translated following the translation made by CC.
```suggestion
* [Creative Commons Attribuzione 4.0 Internazionale](#cc-attribution-40-international)
```